### DISCUSSION TITLE
Can ACME.sh run on mainframe USS?

### DISCUSSION THREAD

**Discussion by dickywmc** [acmesh-official/acme.sh#5298] [Q&A]
I am trying to renew mainframe certificates using ACME.sh and plan to run ACME.sh under mainframe USS (Unix System Services).  From README, the way to install ACME.sh was vig GIT CLONE, and to install GIT I need to first install Miniconda from Rocket software.  The miniconda package fails to run, saying there is not enough memory.

May I know if anyone has tried to install and run ACME.sh on the mainframe and can share with me how this can be done?

Any help will be greatly appreciated!  Thanks.

**Comment by neilpang** [comment]
you don't need git.  just download the tarball. and run:
```
./acme.sh --install
```

**Reply by dickywmc** [reply]
Thanks Neil.  Do you mind elaborating a bit how to "download the tarball"?  Sorry that I am new to GIT, my experience is mostly on the mainframe.  Thanks!

P.S. FTP is working on the mainframe so I can transfer anything from my laptop to mainframe.

**Comment by neilpang** [comment]
ask google for "download tarball from github"
